modules/gutenberg/module.php

```diff
@@ -73,7 +73,19 @@ public function enqueue_assets() {
 
 		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
 
-		wp_enqueue_script( 'elementor-gutenberg', ELEMENTOR_ASSETS_URL . 'js/gutenberg' . $suffix . '.js', [ 'jquery' ], ELEMENTOR_VERSION, true );
+		wp_enqueue_script( 'elementor-gutenberg', ELEMENTOR_ASSETS_URL . 'js/gutenberg' . $suffix . '.js',
+			[
+				'jquery',
+			],
+		ELEMENTOR_VERSION, true );
+
+		wp_enqueue_script( 'elementor-ai-gutenberg', ELEMENTOR_ASSETS_URL . 'js/ai-gutenberg' . $suffix . '.js',
+			[
+					'jquery',
+					'elementor-v2-ui',
+					'elementor-v2-icons',
+			],
+		ELEMENTOR_VERSION, true );
```

Comment:
Please add AI config and translation domain:

https://github.com/elementor/elementor/blob/0585ab7a08faec5b8a513a5ecc5fba0754b41648/modules/ai/module.php#L146-L169

Reply:
Let's go over it together